code/processes/wdb.q

```diff
@@ -203,27 +203,32 @@ endofdaysave:{[dir;pt]
 	.lg.o[`savefinish;"finished saving data to disk"];
 	};
 
-/- add entries to dictionary of callbacks. if timeout has expired or d now contains all expected rows then it releases each waiting process
+/- add entries to table of callbacks. if timeout has expired or d now contains all expected rows then it releases each waiting process
 handler:{
-	.wdb.d[.z.w]:x;
-	if[(.proc.cp[]>.wdb.timeouttime) or (count[.wdb.d]=.wdb.countreload);
-		.lg.o[`handler;"releasing processes"];
-		.wdb.flushend[];
-		.wdb.d:()!()];
-	};
+	/-insert process reload outcome into .wdb.reloadsummary 
+                .wdb.reloadsummary[.z.w]:.wdb.reloadstatus;
```

Comment:
what is .wdb.reloadstatus? 
Surely this should be the inbound parameter?

Reply:
.wdb.reloadstatus is defined in the .wdb.reloadproc function at the top
https://github.com/AquaQAnalytics/TorQ/blob/137521af9d0cf051fa8be64b9032fbdc12a3dfef/code/processes/wdb.q#L380-L381
If the process reload fails I have then used an async set here:
https://github.com/AquaQAnalytics/TorQ/blob/137521af9d0cf051fa8be64b9032fbdc12a3dfef/code/processes/wdb.q#L380-L381

Reply:
Then the handler is called here:
https://github.com/AquaQAnalytics/TorQ/blob/137521af9d0cf051fa8be64b9032fbdc12a3dfef/code/processes/wdb.q#L389-L392

And .wdb.reloadstatus should be set the correct values to be inserted to .wdb.reloadsummary here:
https://github.com/AquaQAnalytics/TorQ/blob/137521af9d0cf051fa8be64b9032fbdc12a3dfef/code/processes/wdb.q#L209

I pushed these changes in order to get away from the multiple handler calls if a process reload fails discussed earlier